src/screens/swap/swap-form/swap-route-item/swap-route-item-icon/swap-route-item-icon.tsx

```diff
@@ -9,13 +9,14 @@ import { isDefined } from '../../../../../utils/is-defined';
 
 interface SwapRouteTokenIconProps {
   tokenSlug: string;
+  size?: number;
 }
 
-const SwapRouteTokenIcon: FC<SwapRouteTokenIconProps> = ({ tokenSlug }) => {
+const SwapRouteTokenIcon: FC<SwapRouteTokenIconProps> = ({ tokenSlug, size = 24 }) => {
```

Comment:
`formatSize` function is usually applied externally and also to the default size (see https://github.com/madfish-solutions/templewallet-mobile/blob/f323342a71af3a453bb3f11f581686a02bf8df31/src/components/token-dropdown/token-dropdown-item/token-dropdown-item.tsx#L29 for instance). It would be better to follow this practice here as well